build_environment.yml

```diff
@@ -37,10 +37,12 @@ dependencies:
   - requests
   - setuptools
   - six
+  - sphinx
   - trollimage
   - trollsift
   - xarray
   - scipy
   - pip:
     - git+https://github.com/pytroll/satpy.git@master
     - graphviz
```

Comment:
@djhoese Do we need graphviz in pip and conda?

Reply:
iirc one is the C program (from conda) and one is the python library (pip) that is used in sphinx...I think.